.github/workflows/lint.yml

```diff
@@ -27,7 +27,10 @@ jobs:
           pip install black
           black --check ./aws/logs_monitoring
 
-      - name: Lint CloudFormation template with cfn-lint
-        uses: scottbrenner/cfn-lint-action@master
-        with:
-          args: "aws/logs_monitoring/template.yaml"
+      - name: Setup Cloud Formation Linter with Latest Version
```

Comment:
Looks like this doesn't necessarily use the latest version (it will always use version 2). I would just stick to "Setup Cloud Formation Linter"

Reply:
It's v2 of the github action, but I think it does use the latest version of the linter, at least based on the actions documentation. https://github.com/ScottBrenner/cfn-lint-action#upgrading-from-version-1